code/obj/critter/misc.dm

```diff
@@ -443,7 +443,7 @@
 
 /obj/critter/spacescorpion
 	name = "space scorpion"
-	desc = "A scorpion in space."
+	desc = "A scorpion in space, it might be less angry if you offer it something..."
```

Comment:
Not super in kahoots with the phrasing here, might I humbly suggest:

```suggestion
	desc = "A scorpion in space. It seems a little hungry,"
```

Reply:
ooh, more subtle